examples/account/README.md

```diff
@@ -1,118 +1,154 @@
-# Aiven Project, Account, and Teams Example
+# Aiven organizations, units, and projects example
```

Comment:
this has to be created separately from `examples/account`, e.g. as `examples/organization`, let's not delete the old docs while the resources are still partially supported